.github/workflows/docker-v2-release.yml

```diff
@@ -0,0 +1,277 @@
+name: Docker v2 release
+
+on:
+  pull_request: # TODO: Remove after testing the PR
+  release:
+    types: [published]
+  push:
+    branches:
+      - release\/*
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+env:
+  TMP_LOCAL_IMAGE: localhost:5000/radixdlt/babylon-node-test
+  REGISTRY_IMAGE: radixdlt/babylon-node-test # TODO: Rename to babylon-node after testing the PR
+
+jobs:
+  setup_version_properties:
+    name: Setup version properties
+    runs-on: ubuntu-latest
+    outputs:
+      VERSION_BRANCH: ${{ steps.setup_version_properties.outputs.VERSION_BRANCH }}
+      VERSION_BUILD: ${{ steps.setup_version_properties.outputs.VERSION_BUILD }}
+      VERSION_COMMIT: ${{ steps.setup_version_properties.outputs.VERSION_COMMIT }}
+      VERSION_DISPLAY: ${{ steps.setup_version_properties.outputs.VERSION_DISPLAY }}
+      VERSION_TAG: ${{ steps.setup_version_properties.outputs.VERSION_TAG }}
+      VERSION_LAST_TAG: ${{ steps.setup_version_properties.outputs.VERSION_LAST_TAG }}
+    steps:
+      - uses: RDXWorks-actions/checkout@main
+        with:
+          fetch-depth: 0
+      - name: Setup version properties
+        id: setup_version_properties
+        uses: ./.github/actions/setup-version-properties
+
+  setup_tags:
+      name: Setup Docker tags
+      runs-on: ubuntu-latest
+      outputs:
+        tag: ${{ steps.setup_tags.outputs.tag }}
+      steps:
+        - uses: RDXWorks-actions/checkout@main
+          with:
+            fetch-depth: 0
+        - id: setup_tags
+          run: echo "tag=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
+  build_deb:
```

Comment:
I appreciate it's not related to this change, but would it be easy to move this out to a separate workflow, e.g. `deb-package.yml`?

It's not related to docker, and confusing it's in here... You'll need to copy setup_version_properties across to it too.